### ISSUE TITLE
add Keycloak

### ISSUE THREAD

**Issue by MishaVeldhoen** [awesome-selfhosted/awesome-selfhosted#2543]
Would [keycloak](https://www.keycloak.org/), be a good project to add to this list?

**Comment by kokomo123** [comment]
Should we toss this in Misc/Other? Federated Identity/Authentication points to awesome-sysadmin, but not sure if they are doing PRs atm.

**Comment by nodiscc** [comment]
It should go to https://github.com/awesome-foss/awesome-sysadmin/tree/9933cce7882e1670c7fe38fda0d98044624c3195#identity-management but we are in the middle of transferring the repo, finding a maintenance workflow/adding tools, etc (https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2482). On hold until we sort it out

**Comment by terribleplan** [comment]
Ultimately I just want to see one list of stuff I can actually grab and run here, and Keycloak and a few other IDPs certainly fit that description... not sure what is going on in the background but incorporating other "awesome" seems less than ideal as items in their list may or may not be selfhosted. Sorry if this isn't the right venue for discussion that may be covered by "what is going on in the background".

**Comment by nodiscc** [comment]
The planned/work-in-progress system will allow us to have Keycloak both in awesome-sysadmin and awesome-selfhosted

> Ultimately I just want to see one list of stuff I can actually grab and run here, and Keycloak and a few other IDPs certainly fit that description...

I get it

**Comment by nodiscc** [comment]
Blocked by https://github.com/awesome-selfhosted/awesome-selfhosted/issues/1038

**Comment by nodiscc** [comment]
Please resubmit it to https://github.com/awesome-foss/awesome-sysadmin#identity-management
